cmd/commands.go

```diff
@@ -346,6 +345,41 @@ EXPERIMENTAL: May be changed or removed.
 		}(),
 	})
 
+	RegisterCommand(Command{
+		Name: "manpage",
```

Comment:
This is cool, how do I actually use this on my system so I can do `man caddy` and have it show me the manpages?

Reply:
My understanding is it would look something like this (not tested):

```
caddy manpage | sudo tee /usr/share/man/man1/caddy && sudo mandb
```

But really, package installers will do this on install/update. I'll probably update the package install script for the apt repo after playing around with it for a while.

Reply:
Confirmed this isn't quite right, I assumed `caddy manpage` would spit out to stdout but it takes a `--directory` instead. But this is probably pretty close to being correct.

Reply:
This worked:
```
$ mkdir man
$ ./caddy manpage --directory man
$ gzip -r man/
$ sudo cp man/* /usr/share/man/man8/
$ sudo mandb
```

Will need to add this to the deb package installation later.